<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(85, 170, 255);
}
QTableWidget{
	color: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_Name">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
     border-width: 1px;
     border-radius: 6px;

</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Password">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
     border-width: 1px;
     border-radius: 6px;

</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Repassword">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
     border-width: 1px;
     border-radius: 6px;
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;
background-color: rgb(115, 115, 115);
border-top:0px solid #e8f3f9;
background:  transparent;</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Password">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;
background-color: rgb(115, 115, 115);
border-top:0px solid #e8f3f9;
background:  transparent;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Repassword">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;
background-color: rgb(115, 115, 115);
border-top:0px solid #e8f3f9;
background:  transparent;</string>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mail">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>280</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;幼圆&quot;;
background-color: rgb(115, 115, 115);
border-top:0px solid #e8f3f9;
background:  transparent;</string>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Mail">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: outset;
     border-width: 1px;
     border-radius: 6px;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Register">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 9pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);
 border-style: outset;
     border-width: 1px;
     border-radius: 6px;
     border-color: beige;    
     padding: 6px;
	background-color: rgb(80, 160, 240);
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
                   }
QPushButton:hover{
	
	background-color: rgb(85, 170, 255);
                   border-color:rgba(255,255,255,200);
                   
	color: rgb(255, 255, 255);
                   }</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Back">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>93</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 9pt &quot;幼圆&quot;;
color: rgb(255, 255, 255);
 border-style: outset;
     border-width: 1px;
     border-radius: 6px;
     border-color: beige;    
     padding: 6px;
	background-color: rgb(80, 160, 240);
}
QPushButton:pressed{
                   background-color:rgba(100,255,100,200);
                   border-color:rgba(255,255,255,30);
                   border-style:inset;
                   color:rgba(0,0,0,100);
                   }
QPushButton:hover{
	
	background-color: rgb(85, 170, 255);
                   border-color:rgba(255,255,255,200);
                   
	color: rgb(255, 255, 255);
                   }</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>421</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/icon/zhuce.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>tableWidget</zorder>
  <zorder>lineEdit_Name</zorder>
  <zorder>lineEdit_Password</zorder>
  <zorder>lineEdit_Repassword</zorder>
  <zorder>label_Name</zorder>
  <zorder>label_Password</zorder>
  <zorder>label_Repassword</zorder>
  <zorder>label_Mail</zorder>
  <zorder>lineEdit_Mail</zorder>
  <zorder>pushButton_Register</zorder>
  <zorder>pushButton_Back</zorder>
  <zorder>label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
